src/cpp/session/modules/connections/SessionConnections.cpp

```diff
@@ -585,6 +592,95 @@ bool isSuspendable()
    return activeConnections().empty();
 }
 
+Error installOdbcDriver(const json::JsonRpcRequest& request,
+                        json::JsonRpcResponse* pResponse)
+{
+   // get parameters
+   std::string driverName;
+   std::string installPath;
+   Error error = json::readParams(request.params, &driverName, &installPath);
+   if (error)
+      return error;
+
+   // find the tools module
+   FilePath rPath = session::options().coreRSourcePath();
+   std::string toolsPath = core::string_utils::utf8ToSystem(
+      rPath.childPath("Tools.R").absolutePath());
+
+   // find connection installer module
+   FilePath modulesPath = session::options().modulesRSourcePath();
+   std::string scriptPath = core::string_utils::utf8ToSystem(
+      modulesPath.complete("SessionConnectionsInstaller.R").absolutePath());
+
+   // source the command
+   std::string cmd;
+   cmd.append("source('");
+   cmd.append(string_utils::jsLiteralEscape(toolsPath));
```

Comment:
`jsLiteralEscape` is for JavaScript -- I think we want `string_utils::singleQuotedStrEscape` here.

Reply:
Got this from... https://github.com/rstudio/rstudio/blame/54cd3abcfc58837b433464c793fe9b03a87f0bb4/src/cpp/session/modules/SessionUpdates.cpp in case you want to fix that one too.

Reply:
Changed.